src/fs/fields.rs

```diff
@@ -194,8 +194,8 @@ pub enum Size {
 /// - <http://www.lanana.org/docs/device-list/devices-2.6+.txt>
 #[derive(Copy, Clone)]
 pub struct DeviceIDs {
-    pub major: u8,
-    pub minor: u8,
+    pub major: i64,
+    pub minor: i64,
```

Comment:
Considering libc defines `pub type c_uint = u32;`, I don't see the reason for using `i64`, would this ever be negative?  https://docs.rs/libc/latest/libc/type.c_uint.html

Reply:
this might be to make space for this difference: 

```rust
# Linux
pub type dev_t = u64;
pub fn major(dev: ::dev_t) -> ::c_uint;
pub fn minor(dev: ::dev_t) -> ::c_uint;

# macOS
pub type dev_t = i32;
pub fn major(dev: dev_t) -> i32;
pub fn minor(dev: dev_t) -> i32;
```

https://github.com/ogham/exa/issues/1126#issuecomment-1273268707

Reply:
> would this ever be negative?

For glibc, I guess it won't be.

Ref: [how to interpret the `dev_t` type](https://stackoverflow.com/a/73923376/14092446)

Reply:
> > would this ever be negative?
> 
> For glibc, I guess it won't be.
> 
> Ref: [how to interpret the `dev_t` type](https://stackoverflow.com/a/73923376/14092446)

I guess the original intent might have been to have space enough for both `pub fn major(dev: dev_t) -> i32;` and `pub fn major(dev: ::dev_t) -> ::c_uint;` (u32)?